src/controllers/post.js

```diff
@@ -20,6 +21,30 @@ export const create = async (req, res) => {
   return sendDataResponse(res, 201, { post: createdPost })
 }
 
+export const createComment = async (req, res) => {
+  const { content, postId } = req.body
+
+  if (!content) {
```

Comment:
I think you should validate the `postId` here as well - if it is not provided, return a 400 just like you do if content is not provided. Can you also add a check for the content length? If it is over 150, return a 400 error too.